CrewChiefV4/Events/OvertakingAidsMonitor.cs

```diff
@@ -62,20 +64,28 @@ override protected void triggerInternal(GameStateData previousGameState, GameSta
                 }
                 if (currentGameState.SessionData.IsNewLap)
                 {
-                    if (drsAvailableOnThisLap && !hasUsedDrsOnThisLap)
+                    if ((drsAvailableOnThisLap && !hasUsedDrsOnThisLap)
+                        || (CrewChief.gameDefinition.gameEnum == GameEnum.RF2_64BIT && drsAvailableInAnyZoneOnThisLap && !hasUsedDrsOnThisLap))
                     {
                         audioPlayer.playMessage(new QueuedMessage("missed_available_drs", MessageContents(folderDontForgetDRS), 0, this));
                     }
                     drsAvailableOnThisLap = currentGameState.OvertakingAids.DrsAvailable;
+                    drsAvailableInAnyZoneOnThisLap = currentGameState.OvertakingAids.DrsAvailable;
```

Comment:
I wonder if instead of all the special casing I did for rF2, I remove it all, but simply set drsAvailableOnThisLap to true if DrsAvailable ever becomes true?